src/plugins/intel_cpu/src/emitters/snippets/x64/jit_brgemm_emitter.hpp

```diff
@@ -8,6 +8,8 @@
 #include "emitters/snippets/x64/kernel_executors/brgemm_base.hpp"
 #include "jit_binary_call_emitter.hpp"
 
+using namespace ov::intel_cpu::x64;
```

Comment:
Please avoid `using namespace` usage in headers

Reply:
removed from headers, thanks Vladislav!